raft/progress.go

```diff
@@ -328,10 +328,37 @@ func (p *prs) committed() uint64 {
 }
 
 func (p *prs) removeAny(id uint64) {
+	pN := p.nodes[id]
+	pL := p.learners[id]
+
+	switch {
```

Comment:
Just use `if` instead of this `switch` idiom.

Reply:
Done.